biapol_utilities/label/_compare_labels.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from sklearn.metrics import jaccard_score
+import pandas as pd
+
+
+def compare_labels(label_image_x, label_image_y):
+    """
+    Compares two label images label_image_x and label_image_y regarding the
+    label-wise Jaccard- and Dice scores.
+    The Jaccard-score is defined as the intersection over union of two labelled
+    images.
+    The Dice score S can be derived from the Jaccard-score J through the
+    following relation:
+    .. math:: S = \frac{2J}{1+J}.
+
+    Parameters
+    ----------
+    label_image_x : ND-array, int
+        Ground truth label image of arbitrary dimensions.
+    label_image_y : ND-array, int
+        Predicted label image which will be compared to label_image_x.
+
+    Returns
+    -------
+    pandas dataframe with columns `[label, jaccard_score, dice_score]. Each row
+    corresponds to the measured quantitiy (Jaccard- or Dice score) of the
+    respective label.
+    """
+
+    # Flatten input
```

Comment:
Can you please write WHY what is done? "Flatten" is a term that means something different in ImageJ

Reply:
Added this to the comment and replaced `flatten()` with `ravel()`:

```python
  # Convert image to 1D array. Makes elemt-wise comparison of arrays simpler.
  label_image_x = label_image_x.ravel()
  label_image_y = label_image_y.ravel()
```